### PR TITLE
newdevice: Updated powerwalker sensors to use custom mib + new feature of sensor discovery for options

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

This will need a notification sending out.

So moved powerwalker to use it's own mib rather than the bodge at rfc compat.

Added support to sensor discovery in yaml to provide group options for sensor classes to avoid code duplication.

### CODE REVIEW

**Comment by laf** [comment]
@murrant If this gets approved, I'd like to take the code changes and put them into a new PR so we can make use of them in other current PRs rather than waiting for this to be merged after a notification goes out.

File: doc/Developing/os/Health-Information.md [hunk 1]
```diff
@@ -40,23 +40,28 @@ mib: NETBOTZV2-MIB
...
         airflow:
-            -
-                oid: airFlowSensorTable
-                value: airFlowSensorValue
-                divisor: 10
-                num_oid: .1.3.6.1.4.1.5528.100.4.1.5.1.2.
-                descr: airFlowSensorLabel
-                index: 'airFlowSensorValue.{{ $index }}'
+            data:
+                options:
```
Comment (murrant) [inline doc/Developing/os/Health-Information.md]: options is in the wrong place here.

Comment (murrant) [inline doc/Developing/os/Health-Information.md]: This options is in the wrong place.

**Comment by murrant** [comment]
Looks good overall!  How much of a gain is it to have the group options vs just requiring them in each sensor?

**Comment by laf** [comment]
This pr would have meant having them in 90% of the sensors so just seems daft not to make it more global. Divisor is the same for a lot of devices so I think it's a good enough saving.

**Comment by murrant** [comment]
Sounds good to me.  Go ahead and split that off (fixing the documentation with invalid yaml)

**Pull request closed without merging**